pkg/controller/node/node_controller.go

```diff
@@ -130,6 +130,10 @@ func (r *ReconcileNode) syncLabels(name string, labels map[string]string) error
 		return nil
 	}
 
+	if r.stosClient == nil {
+		return ErrNoAPIClient
+	}
+
```

Comment:
Thanks. We need to add more tests for the functions in each of the controllers. `pkg/controller/storageoscluster/storageoscluster_controller_test.go` has some tests for the cluster controller.